examples/archive.html

```diff
@@ -104,17 +111,20 @@ <h3 id="offcanvasRightLabel">Images</h3>
                 let imageRow = document.createElement('div');
                 let image = new Image(150, 150);
                 let placeButton = document.createElement('a');
-
+                let imageArray=[];
                 placeButton.classList.add('btn', 'btn-sm', 'btn-outline-secondary', 'place-button');
                 placeButton.innerHTML = 'Place on map';
-
+                let imageGroup = L.distortableCollection().addTo(map)
                 image.src = `${url.replace('metadata', 'download')}/${file.name}`;
-
                 imageRow.classList.add('d-flex', 'justify-content-between', 'align-items-center', 'mb-4', 'pe-5');
                 imageRow.append(image, placeButton);
-
+                imageArray.push[image]
                 imageContainer.appendChild(imageRow);
                 imageCount++;
+                
+                for(let i=0; i<imageArray.length;i++){
```

Comment:
And likewise, here maybe we don't need to iterate with a for loop - we can just run `imageGroup.addLayer(image[i])` down on line 141 when the user clicks "Place" -- does that make sense?